cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -275,6 +275,15 @@ impl pallet_assets::Config<TrustBackedAssetsInstance> for Runtime {
 	type BenchmarkHelper = ();
 }
 
+// Allow Freezes for the `Assets` pallet
+pub type TrustBackedAssetsFreezerInstance = pallet_assets_freezer::Instance1;
```

Comment:
For consistency with other type names

```suggestion
pub type AssetsFreezerInstance = pallet_assets_freezer::Instance1;
```

Reply:
actually, what's the purpose of this type aliases? they don't seem to be used anywhere. if that's the case, maybe we just remove them?